### ISSUE TITLE
[Feature Request] Timeline Visualization compatibilty

### ISSUE THREAD

**Issue by alx-xlx** [vrtmrz/obsidian-livesync#316]
I think it would be super cool to visualize the timeline of the obsidian brain with something like [gource](https://gource.io/).

[acaudwell/Gource](https://github.com/acaudwell/Gource)

At the moment we are just adding data to obsidian without any time and user/device tracking. I don't think it would be that difficult as it just needs a log file.

OR are their other ways to achieve the same.

**Comment by dlphnkck** [comment]
Hi! Are you making a mistake? Are these the Self-hosted LiveSync plugin issues?
What does this have to do with the LiveSync plugin?

**Comment by alx-xlx** [comment]
> Hi! Are you making a mistake? Are these the Self-hosted LiveSync plugin issues? What does this have to do with the LiveSync plugin?

You are right its not a Obsidian-Livesync issue, It's a feature request you can clearly see that in the title, I think "Self-hosted LiveSync plugin" is the perfect plugin for this purpose, as it keeps a record of all updates to markdown files from different devices. All it needs is to generate a separate log file with gource readable syntax.

And with this log file/files anyone can run the gource on their system to generate the visualization.
